libs/blocks/marquee/marquee.css

```diff
@@ -203,6 +203,20 @@
   object-position: left bottom;
 }
 
+.marquee .text .icon-area.icon-area-multiple {
```

Comment:
I think this can be simplified a bit to save a few bytes and still work as needed:

```suggestion
.marquee .icon-area-multiple {
```

Reply:
I simplified it to:

`.marquee .icon-area.icon-area-multiple`

because the styles from `marquee.split .icon-area {` are overriding this (even if the css is above the file) so this won't work without putting `.icon-area` next to `.icon-area-multiple`